build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-# install dependencies with yarn
-yarn install
+# install dependencies
+npm install
```

Comment:
We do not want to use another package manager. `npm` is enough for this project.